src/main/java/app/entities/unitconverter/VolumeConverter.java

```diff
@@ -0,0 +1,28 @@
+package app.entities.unitconverter;
+
+public class VolumeConverter {
+
+    public double cubicMeterToLiter(double cubicMeters) {
+        return cubicMeters * 1000;
+    }
+    public double cubicMeterToMilliliter(double cubicMeters) {
+        return cubicMeters * 1000000;
+    }
+
```

Comment:
inconsistencies in newlines. Please use 1 new line between each method,

Reply:
Fixed it, only 1 new line between each method